### PR TITLE
helper scripts for running test and deploying topo

### PR DESCRIPTION
**Pull request by andriidovhan**
added helper script:

ansible/hedgehog_deployment_script.sh - remove / add topo, deploy minigraph
tests/hedgehog_test_runner.py - custom runner, for running tests from tests/hedgehog_test_list.yaml

### CODE REVIEW

**Review by nazar-garmadiy — APPROVED** [review]

File: ansible/hedgehog_deployment_script.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  echo "Pre check"
+  if [ -z "$MGMT_CONTAINER" ]; then
+    echo "No sonic-mgmt container is running. Exit 1."
+    exit 1
+  else
+    echo "The sonit-mgmt container is running: $MGMT_CONTAINER"
+  fi
+}
+
+remove_topo () {
```
Comment (maksymhedeon) [inline ansible/hedgehog_deployment_script.sh:20]: There is `veos-vm/cleanup.sh` script. Should we use it here?

Comment (andriidovhan) [reply ansible/hedgehog_deployment_script.sh:20]: didn't know about that script, 
I just followed the instruction
https://github.com/githedgehog/sonic-mgmt/blob/master/docs/testbed/README.testbed.VsSetup.md#restoreremove-the-testing-environment

Comment (andriidovhan) [reply ansible/hedgehog_deployment_script.sh:20]: I just checked this script, it is do more than we need.
I mean this script stop all VMs, remove VM disks, stop/remove all containers/images

I propose do not use this script. just to save time for deploying

Comment (andriidovhan) [reply ansible/hedgehog_deployment_script.sh:20]: extend this script with `cleanup.sh` in separate pr

**Pull request merged**